indico/modules/users/util.py

```diff
@@ -206,8 +207,7 @@ def build_user_search_query(criteria, exact=False, include_deleted=False, includ
             raise ValueError("'name' is not compatible with 'exact'")
         if 'first_name' in criteria or 'last_name' in criteria:
             raise ValueError("'name' is not compatible with (first|last)_name")
-        name = name.split()
-        query = query.filter(_build_name_search(name))
+        query = query.filter(_build_name_search(name.replace(',', '').split()))
```

Comment:
What if someone types 'Monnich,Adrian' (no space)? It would end up being 'MonnichAdrian'. I wonder if `ilike` is able to find 'Monnich__Adrian' (two spaces), so we could safely do `name.replace(',', ' ')`.

Reply:
I thought about it as well, but in the end I thought "garbage in, garbage out". If someone enters a name in a way nobody uses.. their fault ;)